README.md

```diff
@@ -4,3 +4,5 @@ Hello!
 
 This is a simple repository to demonstrate Git and Github usage for COMP4530, 
 Advanced Software Engineering at Thompson Rivers University.
+
+:chicken:
```

Comment:
Not sure that I agree that it should be a :cow:, I feel that :chicken: better conveys the mandatory level of business synergy